manifests/mod/remoteip.pp

```diff
@@ -51,17 +51,17 @@
 #   `mod_remoteip`. If not specified, `$::apache::apache_version` is used.
 #
 class apache::mod::remoteip (
-  String                         $header                    = 'X-Forwarded-For',
-  Optional[Array[Stdlib::Host]]  $internal_proxy            = undef,
-  Optional[Array[Stdlib::Host]]  $proxy_ips                 = undef,
-  Optional[Stdlib::Absolutepath] $internal_proxy_list       = undef,
-  Optional[String]               $proxies_header            = undef,
-  Boolean                        $proxy_protocol            = false,
-  Optional[Array[Stdlib::Host]]  $proxy_protocol_exceptions = undef,
-  Optional[Array[Stdlib::Host]]  $trusted_proxy             = undef,
-  Optional[Array[Stdlib::Host]]  $trusted_proxy_ips         = undef,
-  Optional[Stdlib::Absolutepath] $trusted_proxy_list        = undef,
-  Optional[String]               $apache_version            = undef,
+  String                                                     $header                    = 'X-Forwarded-For',
+  Optional[Array[Variant[Stdlib::Host,Stdlib::IP::Address]]] $internal_proxy            = undef,
```

Comment:
Would `Variant[Stdlib::Host,Stdlib::IP::Address::V4::CIDR,Stdlib::IP::Address::V6::CIDR]` better show intent?
`Stdlib::Host` already allows IP addresses (oddly via `Stdlib::Compat::Ip_address` instead of `Stdlib::IP::Address::Nosubnet`).  The overlap doesn't really matter, and someone should probably submit a `Stdlib::IP::Address::CIDR` type.

Reply:
Would you prefer that I change this to your suggestion? Or should we leave this as-is?